yarn.lock

```diff
@@ -9,6 +9,14 @@
   dependencies:
```

Comment:
this diff is bigger than I’d like it to be. Jest is sorta notorious for causing TS issues, so we should def check everything looks right before merging